src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/ImageLoader.java

```diff
@@ -0,0 +1,186 @@
+package com.microsoft.maui;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.widget.ImageView;
+
+import androidx.annotation.ColorInt;
+import androidx.annotation.Nullable;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.RequestBuilder;
+import com.bumptech.glide.RequestManager;
+import com.bumptech.glide.load.DataSource;
+import com.bumptech.glide.load.engine.DiskCacheStrategy;
+import com.bumptech.glide.load.engine.GlideException;
+import com.bumptech.glide.request.RequestListener;
+import com.bumptech.glide.request.target.Target;
+import com.microsoft.maui.glide.fontimagesource.FontModel;
+
+import java.io.File;
+import java.io.InputStream;
+
+public class ImageLoader {
+    public static void loadFromResourceId(ImageView imageView, int resourceId, ImageLoaderCallback callback)
+    {
+        imageView.setImageResource(resourceId);
+        callback.onComplete(true, null, null);
+    }
```

Comment:
Could this one not have a callback at all? Then the calling code would just assume it's synchronous and finished.

Reply:
Oh if this is synchronous, you can probably delete this method and just call `ImageView`.

Reply:
Removed this.  I was already bypassing it but forgot one place.

So really, testing loading a file is kind of unnecessary for resource drawable files since they'll never hit this code path now, as we check for the drawable reasource id by filename and use `imageView.SetImageResourceId(..)` with it instead of calling out to this Image Loader.

It's really just files elsewhere on disk that will hit this path now I think, and fonts, and urls.